packages/web/src/SplunkErrorInstrumentation.ts

```diff
@@ -30,7 +30,7 @@ import { diag } from '@opentelemetry/api'
 const STACK_LIMIT = 4096
 const MESSAGE_LIMIT = 1024
 
-export const STACK_TRACE_URL_PATTER = /([\w]+:\/\/[^\s/]+\/[^\s?:#]+)/g
+export const STACK_TRACE_URL_PATTER = /([\w]+:\/\/[^\s/]+\/[^\s?:#]+(\?\S[^)\s*]*)?)/g
```

Comment:
To have consistency with cli tool, let us use 
`export const STACK_TRACE_URL_PATTER =/[\w]+:\/\/[^\s]+?(?::\d+)?(?=:[\d]+:[\d]+)/g`

Reply:
Alright. So, just change this one line in this PR?